README.md

```diff
@@ -86,9 +86,37 @@ Add a new tab to the tab group and returns a `Tab` instance.
 * `active` (default: `false`): set this to `true` if you want to activate the tab once it is loaded. Otherwise you will need to call `tab.activate()`.
 * `ready`: a callback function to call once the tab is ready. The `Tab` instance is passed as the only parameter.
 
-#### `tabGroup.getTab(id)`
+#### `tabGroup.getTab(id, idIsPosition)`
 
-Retrieve an instance of `Tab` from its `id` (return `null` if not found).
+If `idIsPosition` is true `id` is passed to `tabGroup.getTabByPosition`. Otherwise, `id` is passed to `tabGroup.getTabById`.
+
+#### `tabGroup.getTabById(id)`
+
+Retrieve an instance of `Tab` from this `id` (return `null` if not found).
+
+#### `tabGroup.getTabByPosition(position)`
+
+Retrieve an instance of `Tab` from this `position` (return `null` if not found). A negative value is an offset from the right.
+
+To get the tab in the leftmost position:
+
+```javascript
+tabGroup.getTabByPosition(1);
+```
+
+To get the tab in the rightmost position:
+
+```javascript
+tabGroup.getTabByPosition(-1);
+```
+
+> Note: Position 0 does not contain a tab.
```

Comment:
😮 I totally missed this detail when I merged #43!! First tab should be index 0, because it is the standard way to do in javascript.
My mistake...
I don't want to break the compatibility so let's keep it like this until the next major version is released.

Reply:
I made the leftmost tab index 1 so it would be consistent with the negative indeces. Then, you use positive 1 to get the leftmost and negative 1 to get the rightmost.

Reply:
I understand, but still I think it is really confusing for most programmers. In most computer programming languages (including javascript) the first element has the index 0. I don't think it is relevant to do something different here. Anyway it is too late now, so let's keep it as it is until we change it on next major release.